app/controllers/redirections_controller.rb

```diff
@@ -0,0 +1,13 @@
+class RedirectionsController < ApplicationController
+  def next
+    redirection = Redirection.find_by(slug: params[:slug])
+
+    redirect_to redirection.next.url
+  end
+
+  def previous
```

Comment:
> We're not really dealing with traditional crud resources.

We're dealing with throwback REST resources: REST only dates from 2000, while webrings date from like 1995.

This is fine.